scripts/jenkins/jobs-ibs/cloud-mediacheck.yaml

```diff
@@ -24,6 +24,7 @@
             - SUSE:SLE-11-SP3:Update:Cloud5:Test
             - SUSE:SLE-12-SP1:Update:Products:Cloud6/SLE_12_SP1
             - SUSE:SLE-12-SP2:Update:Products:Cloud7/SLE_12_SP1
```

Comment:
But we actually want to build against SP2 anyway, do you know where/how to change it?